packages/eslint-plugin/tests/docs.test.ts

```diff
@@ -180,6 +181,32 @@ describe('Validating rule docs', () => {
           }
         });
       }
+
+      test('must include only valid code samples', () => {
+        for (const token of tokens) {
+          if (token.type !== 'code') {
+            continue;
+          }
+
+          const lang = token.lang?.trim();
+          if (!lang?.startsWith('ts')) {
+            return;
```

Comment:
Why is this `return` instead of `continue`?

Reply:
Thanks for catching this! It turns out `docs/rules/explicit-function-return-type.md` also had code blocks with syntax errors, but because of this eager return, the tests weren't catching them